build.gradle

```diff
@@ -36,7 +36,7 @@ ext {
     pmdVersion = '5.6.1'
     jacocoVersion = '0.7.9'
     findbugsVersion = '3.0.1'
-    commercetoolsSyncJava='1.9.1'
+    commercetoolsSyncJava='2.0.0'
```

Comment:
I know the dependabot will udpate but it would be nice to check other references like commercetoolsJvmSdk which could be 1.53.0.

Reply:
But this issue is to update sync-java to version 2.0.0, so I would not mix it. And also we have a task for this in our sprint and limited time there :)

Reply:
Normally when you update java-sync you are updating the jvm version too, which we are documented that way, but it's ok changing one line is not a time-consuming thing.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/158/commits/7ce27c6349f0869ad9d2f90c6f9d0b1a8c494f2c